nml/actions/action2layout.py

```diff
@@ -67,7 +67,7 @@ def write(self, file):
         file.newline()
         if len(self.sprite_list) == 0:
             file.print_dwordx(0)  # sprite number 0 == no sprite
-            for i in range(0, 5):
+            for _ in range(0, 5):
```

Comment:
I'm not a fan of this loop. What do you think of `file.print_data(b"\0" * 5)`, or just `file.print_data(b"\0\0\0\0\0")`?

Reply:
It's a dword so each value is written \dx00000000 (in NFO) or 00 00 00 00 (in GRF)

Reply:
Good point. Ignore this idea then.